canonical-web-and-design/cn.ubuntu.com
# Add redirect for /iot to /internet-of-things

## Done

- Add redirect for `/iot` to `/internet-of-things`

## QA

- Visit page at: https://cn-ubuntu-com-497.demos.haus/iot
- Check `/iot` redirects to `/internet-of-things` and `/iot/core` redirects to `/internet-of-things/core` (waiting on this [PR](https://github.com/canonical-web-and-design/cn.ubuntu.com/pull/496) to merge for /core page)

## Issue / Card

[#3891](https://github.com/canonical-web-and-design/web-squad/issues/3891)
